Dockerfile

```diff
@@ -146,3 +147,5 @@ COPY --from=manifest /tmp/BUILD /opt/manageiq/manifest
 
 # Switch USER back to postgres
 USER 26
+
+CMD ["miq-run-postgresql"]
```

Comment:
Should this be entrypoint?  What happens if you set the `--entrypoint` to bash - will it still run this?

Reply:
I tried an entrypoint script that would `exec container-entrypoint`, but the environment wasn't passed along.

Reply:
> What happens if you set the --entrypoint to bash - will it still run this?

No, but it wouldn't run if it were an entrypoint script either.  What use case are you trying to solve?